ocaml/parsing/builtin_attributes.ml

```diff
@@ -676,37 +676,30 @@ let error_message_attr l =
     | _ -> None in
   List.find_map inner l
 
-type property =
-  | Zero_alloc
-
-type check_attribute =
-  | Default_check
-  | Ignore_assert_all of property
-  | Check of { property: property;
-               strict: bool;
+type zero_alloc_attribute =
+  | Default_zero_alloc
```

Comment:
I think just `Default` would be a fine name for this constructor, and more in keeping with the others.  But it's fine to ignore this suggestion or do it in a future PR in the interest of getting this out.

Reply:
Let's keep the longer name, mostly in the interest of getting it out. There is a precedent for it in `lambda`, where attributes like `inline` and `local` have longer names for their default settings.